djangoguestbook/guestbook/views.py

```diff
@@ -2,7 +2,36 @@
 from django.views.generic.edit import FormView
 from google.appengine.api import users
 from guestbook.models import Greeting, DEFAULT_GUESTBOOK_NAME
-from guestbook.forms import PostForm
+from guestbook.forms import PostForm, EditForm
+
+
+class EditGreetingView(FormView):
+	template_name = "guestbook/edit_page.html"
+	form_class = EditForm
+	greeting_id = long(0)
+	guestbook_name = DEFAULT_GUESTBOOK_NAME
+
+	def form_valid(self, form):
+		dict = {
+			'greeting_id': self.__class__.greeting_id,
```

Comment:
không dùng biên của class,
lấy data từ ```form.cleaned_data``` ra dùng
https://docs.djangoproject.com/en/1.7/topics/forms/#the-view